osu.Game.Tests/Visual/SongSelect/TestScenePlaySongSelect.cs

```diff
@@ -739,7 +740,7 @@ public void TestDifficultyIconSelectingForDifferentRuleset()
             AddUntilStep("Check ruleset changed to mania", () => Ruleset.Value.ID == 3);
 
             AddAssert("Selected beatmap still same set", () => songSelect.Carousel.SelectedBeatmapInfo.BeatmapSet.ID == previousSetID);
```

Comment:
Seems like this should use OnlineID? Might be worth going over similar cases in this file again since I'm not sure why some cases are doing `ID == ID` comparisons (e.g. `TestHideSetSelectsCorrectBeatmap`) when this PR seems to have changed them to use `OnlineID`.

Reply:
Have done  a pass of this file to only use `OnlineID` now.